examples/examples/ws_pubsub_with_params.rs

```diff
@@ -63,38 +65,37 @@ async fn main() -> anyhow::Result<()> {
 
 async fn run_server() -> anyhow::Result<SocketAddr> {
 	const LETTERS: &str = "abcdefghijklmnopqrstuvxyz";
-	let server = ServerBuilder::default().build("127.0.0.1:0").await?;
+	let server = ServerBuilder::default().set_backpressure_buffer_capacity(10).build("127.0.0.1:0").await?;
```

Comment:
Nit: not a big deal but I wonder whether it's better just to call it `set_buffer_capacity` (backpressure being implied as a result of a buffer existing and the whole future poll mechanism anyway)

Reply:
yeah I know but @lexnv asked what the buffer was for so I wanted it be extra verbose

Not sure, @lexnv what do you prefer?

Reply:
Yep, I do remember that, I think either option is fine for me. How does `set_backpressure(capacity: u32)` feel to you 🤔 ?

Reply:
what about `set_message_buffer`?

Reply:
I'm ok with `set_message_buffer_capacity` or something like that? "Capacity" is nice because it isn't size (it won't use the amount unless needed)